runatlantis.io/docs/custom-workflows.md

```diff
@@ -176,6 +176,41 @@ Atlantis will need to have the `terragrunt` binary in its PATH.
 If you're using Docker you can build your own image, see [Customization](/docs/deployment.html#customization).
 :::
 
+#### Terragrunt Autoplan Configuration
```

Comment:
Can you call this section Terragrunt 0.19+

Reply:
So this actually applies to Terragrunt 0.18 as well. I think it could be confusing to call it Terragrunt 0.19+ since it really applies to all versions.

Reply:
So is the section above where we tell people to list all the projects obsolete then?

Reply:
I think there still might be a place for the above config. It depends on how you're using Terragrunt, really. If you have just a few modules then listing out each project works well. But if you have a ton of modules (like we do) then it makes more sense to use autoplan.